specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -852,7 +852,7 @@
           "TestRun"
         ],
         "summary": "Get test run file by file name.",
-        "operationId": "TestRun_GetFile",
+        "operationId": "LoadTestRun_GetFile",
```

Comment:
Should this be LoadTestRun_GetTestFile to ensure the right side is symmetric to that of LoadTestAdministration_GetTestFile

Reply:
changed to LoadTestRun_GetTestRunFile as it is for TestRun se